<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Canvas" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>801</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="shapeChoicesScrollArea">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>118</width>
        <height>79</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="circleChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ellipseChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ellipse</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="lineChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="polygonChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Polygon</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="polylineChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Polyline</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rectangleChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="squareChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="textChoice">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
